conduct.md

```diff
@@ -7,16 +7,282 @@
+The more we put into our efforts, the better will be the outcome. This can be
+an amazing opportunity to bond well with the team members and bring positivity
+that will bring long-term benefits for the team.
+
+* **Acceptable Participation (C):**
+  * Participating in Slack discussions
+  * Being open to meeting outside of class time
+  * Asking questions
+  * Sharing one's views without triggering someone else's
+  * Enabling discussions in a polite and pleasant form
+  * Disagreeing in a polite manner if required
+
+* **Unacceptable Participation (U):**
+  * Not participating in Slack Polls
+  * Missing lab sections without consent of the course instructor
+  * Not asking questions or participating in discussions
+  * Not doing your share of the labs
+  * Interrupting a team member with criticism without offering him/her the
+    opportunity to share their views
+  * Passing offensive comments during discussions
+  * Trying to dominate a discussion
+  * Talking about topics entirely irrelevant to the discussions
+
+## Decision
+
+Decisions should be made in the environment that every team member's opinion
+is considered important. They should be made collectively as a team or at
+least 80% of the majority's consensus. All information needed to make a
+decision will be clearly stated to every team member and all concerns should
+be clearly expressed. We will use data, rational knowledge and specific
+experience to make decisions instead of popular opinion. Team members need to
+objectively analyzed the options for the best of the team. Trust and respect
+the decision made in order to move on the project.
+
+* **Unacceptable (U):**
+  * Making decisions without notifying team members.
+  * Question the decision after it is made.
+  * Making a decision based on only popular opinion.
+
+## Postmortems
+
+In the postmortems process of the project we will use proper communication and
+teamwork in order to solve problems and issues. Going through postmortems will
+help us to debug and ensure problems that arise get taken care of smoothly and
+surely.
+
+* **Acceptable Postmortems (C)**
+  * Talk in the first part of lab section to discuss where we are in the project.
+  * Team leaders can lead through this discussion.
+  * Teams will bring up problems and errors that occurred, or what worked well.
+  * After finishing a project teams should be assigned to look after the program.
+  * Work as a team to fix problems.
+  * Use proper communication to talk to one another.
+
+* **Unacceptable (U)**
+  * Do not blame anyone for errors or problems
+  * Do not work alone use your team members as a guide
+  * Do not attempt to change things without talking to others
+
+## Conflicts
+
+In the conflicts section of the project, we deal with conflicts in a
+professional manner by communicating any type of issue/ conflict with any of the
+team leaders, technical leads, or the CEO directly.
+
+* **Acceptable Behavior (C)**
+  * Team Members shall:
```

Comment:
I like this style of prescriptive listing, but other sections do not use the same style. The style should be similar in all sections.